python/ray/serve/scripts.py

```diff
@@ -14,6 +15,20 @@
     DEFAULT_HTTP_HOST,
     DEFAULT_HTTP_PORT,
 )
+from ray.dashboard.modules.serve.schema import ServeApplicationSchema
+from ray.autoscaler._private.cli_logger import cli_logger
+
+
+def log_failed_request(response: requests.models.Response):
+    error_message = (
+        "\nRequest failed. Got response status code "
+        f"{response.status_code} with the following message:"
+        f"\n\n{response.text}"
+    )
+    print(error_message)
```

Comment:
why print and `cli_logger.error`?

Reply:
Good catch, I missed that when I was replacing print statements with the logger. Fixed.